package.json

```diff
@@ -36,6 +36,7 @@
     "tslint": "^5.12.0",
     "tslint-language-service": "^0.9.9",
     "typedoc": "^0.13.0",
+    "typedoc-plugin-external-module-map": "^1.0.0",
```

Comment:
new dev dependency: register [build/test dependency CQ](https://github.com/theia-ide/theia/wiki/Registering-CQs#case-new-or-updated-testbuild-dependency-eg-devdependencies)?

Reply:
I wanted to hear feedback before creating build and test CQ

Reply:
@marcdumais-work ok the CQ is accepted: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19411